src/devices/bus/coco/coco_multi.cpp

```diff
@@ -185,6 +195,14 @@ MACHINE_CONFIG_START(coco_multipak_device::device_add_mconfig)
 	MCFG_COCO_CARTRIDGE_HALT_CB(DEVWRITELINE(DEVICE_SELF, coco_multipak_device, multi_slot4_halt_w))
 MACHINE_CONFIG_END
 
+INPUT_PORTS_START( coco_multipack )
+	PORT_START( "MPI Slot Switch" )
```

Comment:
Please use a terse tag without spaces here - it's easier for utilities to deal with.